app/assets/stylesheets/common/foundation/color_transformations.scss

```diff
@@ -46,6 +46,7 @@ $tertiary-hover: dark-light-choose(
 ) !default;
 
 // tertiary-numbers
+$tertiary-25: dark-light-diff($tertiary, $secondary, 93%, -80%) !default;
```

Comment:
Can you add to styleguide as well? 

Would be nice if this color was used _somewhere_ in core as well.